main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+)
+
+func createPhoneBook(names []string, numbers []string) map[string]string {
+	PhoneBook := make(map[string]string)
+	for i, key := range names {
+		if PhoneBook[key] == "" {
```

Comment:
This will not work with empty names/phone numbers. 

It is more idiomatic to use:

```
if _, found := PhoneBook[key]; found { 
```